docs/estargz.md

```diff
@@ -294,6 +310,60 @@ After the TOC is verified, the snapshotter mounts this layer using the metadata
 During runtime of the container, this snapshotter fetches chunks of regular file contents lazily.
 Before providing a chunk to the filesystem user, snapshotter recalculates the digest and checks it matches the one recorded in the corresponding TOCEntry.
 
+## eStargz image with an external TOC
+
+eStargz supports separating TOC into another image called *TOC image*.
+This type of eStargz is the same as the normal eStargz but doesn't contain TOC JSON file (`stargz.index.json`) in the layer blob and has a special footer.
```

Comment:
It would be helpful to explain the motivation of this.

Reply:
Added the motivation.